pkg/cmd/types.go

```diff
@@ -39,6 +39,10 @@ type ImageInspectorOptions struct {
 	Serve string
 	// Chroot controls whether or not a chroot is excuted when serving the image with webdav.
 	Chroot bool
+	// Local controls whether or not an image should be pulled from remote registry. If set
+	// to true, the image won't be pulled and instead the DstPath will be assumed as
+	// location to scan.
+	Local bool
```

Comment:
The docker client suppose to only download the image if it doesn't exist local or a new version is available